pointgrey_camera_driver/CMakeLists.txt

```diff
@@ -19,14 +19,21 @@ catkin_package(
 # the Point Grey EULA prohibits redistributing the headers or the packages which
 # contains them. Fortunately, we can download the archive directly from their 
 # website during this build process.
-find_library(POINTGREY_LIB flycapture)
+find_library(POINTGREY_LIB NAMES libflycapture.so.2 flycapture)
 if(NOT POINTGREY_LIB)
   message(STATUS "libflycapture not found in system library path")
   include(cmake/DownloadFlyCap.cmake)
   download_flycap(POINTGREY_LIB POINTGREY_INCLUDE_DIR) 
   message(STATUS "libflycapture library: ${POINTGREY_LIB}")
   message(STATUS "libflycapture include: ${POINTGREY_INCLUDE_DIR}")
   include_directories(${POINTGREY_INCLUDE_DIR})  
```

Comment:
I wonder why this isn't needed when the library is found. I guess the default path for the headers does the job, but maybe we should still figure out how to set `POINTGREY_INCLUDE_DIR` properly. ?!